tests/test_get_bitinformation.py

```diff
@@ -215,3 +215,23 @@ def test_get_bitinformation_different_dtypes(rasm, implementation):
 def test_get_bitinformation_dim_list(rasm, implementation):
     bi = xb.get_bitinformation(rasm, dim=["x", "y"], implementation=implementation)
     assert (bi.dim == ["x", "y"]).all()
+
+
+def test_implementations_agree():
+    """Test whether the python and julia implementation retrieve the same results"""
+    ds = xr.tutorial.load_dataset("rasm")
```

Comment:
Does this also work for other tutorial datasets?

Reply:
It should! I tested it locally with the `era5int_uvz`. Are you suggesting, we should test for several datasets here?

Reply:
I think I parametrized the input datasets in some other tests

Reply:
parametrize: dataset name, dim